app/src/main/java/com/medmapper/v33001/dao/MedicineDAO.kt

```diff
@@ -14,4 +14,7 @@ interface MedicineDAO {
 
     @Query("DELETE FROM Medicine")
     suspend fun deleteAll()
-}
\ No newline at end of file
+    
+    @Delete
```

Comment:
Instead of using @query and deleteall, we can use a simple @delete and delete method to make the code more clear and concise.

Source: https://developer.android.com/training/data-storage/room/accessing-data